docs/build-wallets.md

```diff
@@ -30,28 +30,5 @@ that the wallet software is using the latest version of the
 | [Signer](https://www.parity.io/signer/)                            | Live              | Parity        | IOS and Android                                            | Non-custodial |          |
 | [Polkadot-JS](https://polkadot.js.org/apps/#/accounts)             | Live              | Parity        | Browser                                                    | Non-Custodial | Staking  |
 | [Polkadot{.js}](https://github.com/polkadot-js/extension)          | Live              | Parity        | Browser extension                                          | Non-custodial | Staking  |
-| [Polkawallet](https://polkawallet.io/)                             | Live              | Polkawallet   | IOS and Android                                            | Non-custodial | Staking  |
-| [Math Wallet](https://www.mathwallet.net)                          | Live              | Math Wallet   | IOS, Android, Browser extension                            | Non-custodial | Staking  |
-| [Trust Wallet](https://trustwallet.com/)                           | Live              | Trust Wallet  | IOS and Android                                            | Non-custodial |          |
-| [ImToken](https://token.im/)                                       | Live              | ImToken       | IOS and Android                                            | Non-custodial | Staking  |
-| [Ownbit](https://ownbit.io/)                                       | Live              | Ownbit        | iOS and Android                                            | Non-custodial |          |
-| [AirGap](https://airgap.it/)                                       | Live              | AirGap        | IOS, Android, MacOS, Windows and Linux                     | Non-custodial | Staking  |
-| [SafePal](https://www.safepal.io/download)                         | Live              | SafePal       | IOS, Android, and Hardware                                 | Non-custodial |          |
-| [Crypto.com](https://crypto.com/en/index.html)                     | Live              | Crypto.com    | IOS and Android                                            | Non-custodial |          |
-| [Ledger App](https://zondax.ch/kusama.html#overview)               | Live              | Zondax        | Hardware                                                   | Non-custodial |          |
```

Comment:
@laboon is the ledger-app for Polkadot no longer recommended? should I stick with using Parity Signer?